gdal/src/test/scala/geotrellis/contrib/vlm/gdal/GDALRasterSourceProviderSpec.scala

```diff
@@ -38,7 +37,7 @@ class GDALRasterSourceProviderSpec extends FunSpec {
     }
 
     it("should not be able to process a GeoTrellis catalog path") {
-      assert(!provider.canProcess("s3://path/to/my/fav/catalog?layer=fav&zoom=3"))
+      assert(!provider.canProcess("gt+s3://path/to/my/fav/catalog?layer=fav&zoom=3"))
```

Comment:
What happens if I give it `s3://path/to/my/fav/catalog?layer=fav&zoom=3`? Would it still know that it can't process it?

Reply:
@jbouffard no, why GDAL can't read smth from `s3://path/to/my/fav/catalog?layer=fav&zoom=3`? In theory this can be some valid path with smth on that side being understand by GDAL. What is your suggestion? You think `http://domain/to/my/fav/catalog?layer=fav&zoom=3` should not work as well?

Reply:
I'm just wondering if the SPI would pick up on that. Since wouldn't both `GDAL` and `GeoTrellis` `RasterSource` think they can process it?

Reply:
I think it is impossible to dispatch in general (?); the right way is always to require prefix for GeoTrellis avro layers, since GDAL can work with anything that has an array of bytes with proper headers on the other side.